spec/utils/ssl/ca.rb

```diff
@@ -0,0 +1,199 @@
+require 'openssl'
+
+module Utils
+  module SSL
+    class CA
```

Comment:
can we combine and share some of this with spec/utils/ssl.rb?

Reply:
done